wheel/Cargo.toml

```diff
@@ -18,7 +18,7 @@ path = "src/lib.rs"
 chia = { path = "..", features = ["py-bindings"] }
 chia-protocol = { path = "../chia-protocol", features = ["py-bindings"]  }
 clvmr = "=0.2.6"
-pyo3 = { version = "=0.15.1", features = ["extension-module", "multiple-pymethods"] }
+pyo3 = { version = "=0.19.0", features = ["extension-module", "multiple-pymethods"] }
```

Comment:
This one is pinned with `=` unlike the others which use `>=`. Is this intentional?

Reply:
yes, it's deliberate. the `wheel` is the root crate, and it makes sense for it to pin the versions. The dependencies don't really need to. This lets us update one place instead of 3 when we bump the version